docs/content/dagster-plus/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -9,37 +9,60 @@ title: Understanding role-based access control in Dagster+ | Dagster Docs
 Role-based access control (RBAC) enables you to grant specific permissions to users in your organization, ensuring that Dagster users have access to what they require in Dagster+, and no more.
 
 In this guide, we'll cover how RBAC works in Dagster+, how to assign roles to users, and the granular permissions for each user role.
+<Note> All roles are enforced both in Dagster+ and the [GraphQL API](https://docs.dagster.io/concepts/webserver/graphql) </Note>
 
 ---
 
-## Dagster+ user roles
+## Dagster+ Pro user roles
 
-Dagster+ uses a hierarchical model for RBAC, meaning that the most permissive roles include permissions from the roles beneath them. The following user roles are currently supported, in order from the **most** permissive to the **least** permissive:
+Dagster+ Pro employs a flexible approach to user roles and permissions. This system is built on two fundamental concepts:
 
-- Organization Admin
-- Admin
-- Editor
-- Launcher (Pro plans only)
-- Viewer
+1. **Permission scope** - Permissions are context-specific, falling into two main categories:
+   - Organization-wide settings (for example, "Create Teams")
+   - Deployment-specific actions (for example, "Launch and Cancel Backfills")
+2. **Role types** - Dagster+ supports two types of roles:
+   - **Default roles:** Hierarchical roles, based on sensible defaults provided by Dagster.
+   - **Custom roles:** Roles you define with specific sets of permissions to match your organization's needs.
+### Teams
 
-For example, the **Admin** user role includes permissions specific to this role and all permissions in the **Editor**, **Launcher**, and **Viewer** user roles. Refer to the [User permissions reference](#user-permissions-reference) for the full list of user permissions in Dagster+.
+Dagster+ Pro users can create teams of users and assign default permission sets. Refer to the [Managing teams in Dagster+](/dagster-plus/account/managing-users/managing-teams) guide for more info.
 
-### User role enforcement
+--- 
+## Creating custom roles
```

Comment:
Looks like many of the steps in this section and the "Editing" and "Deleting" sections need a period at the end.